maintainers/maintainer-list.nix

```diff
@@ -5030,7 +5030,7 @@
     }];
   };
   ivankovnatsky = {
-    email = "ikovnatsky@protonmail.ch";
+    email = "75213+ivankovnatsky@users.noreply.github.com";
```

Comment:
Mail is meant to be another contact method next to GitHub. Setting this to the github noreply address defeats that purpose.

Reply:
I agree, though my OCD told me that I have a publicly open email in the repo. If you say that I should still provide a valid email I will come with another one. I just thought that if I'm not that active maintainer currently, it might not be that big deal.

Reply:
email is required (see comment at top of maintainer-list.nix). If you are not active, you can also remove yourself from the list (though I hope it wont be necessary).